src/base/bittorrent/session.cpp

```diff
@@ -269,6 +288,8 @@ Session::Session(QObject *parent)
     , m_maxUploadsPerTorrent(BITTORRENT_SESSION_KEY("MaxUploadsPerTorrent"), -1, lowerLimited(0, -1))
     , m_isUTPEnabled(BITTORRENT_SESSION_KEY("uTPEnabled"), true)
     , m_isUTPRateLimited(BITTORRENT_SESSION_KEY("uTPRateLimited"), true)
+    , m_utpMixedMode(BITTORRENT_SESSION_KEY("uTPMixedMode"), m_isUTPEnabled, clampValue(0, 1))
```

Comment:
Well, we say: "tomorrow is another day!". That's what happens in the end: 
1. Neither numbers nor words are meaningless to the user, because there is no available information about it.
2. If we had a user manual (or other source of information, eg. man), we could describe either numeric values or string values. 
3. The use of numbers is much easier from the code POV, i.e. other things being equal, the use of numerical values in the configuration file is preferable.
4. The use of abstract numerical values in the code is a way to unmaintainable code, so we should prefer enums there.

@Chocobo1, what do you say?